app/templates/activity.html

```diff
@@ -1,23 +1,22 @@
 {% extends "__base_ui__.html" %}
 
-{% from "_macros.html" import get_user_avatar_url %}
+{% from "_macros.html" import get_user_avatar_url, render_tab_bar, render_tab_panel %}
+
+{% set tabs = [
+  ('activity', gettext('Activity')),
+  ('connected', gettext('Most Connected')),
+  ('complete', gettext('Most Complete')),
+  ('progress', gettext('My Profile Progress'))
```

Comment:
I'm manually calling `gettext` in the definition here rather than the macro so that our gettext-scraper has an easier time finding these strings.